### ISSUE TITLE
CCAppController.h file not found when user tries to build and run the project.

### ISSUE THREAD

**Issue by khanbhai89** [apportable/BridgeKitSample.spritebuilder#1]
1. Clone the project.
2. Open the Project file.
3. Try to run on iOS Simulator or an Android target after installing the plugin.
   
   -- Result: 'CCAppController.h' file not found error is generated when user tries to run it.
      This happened 3/3 times.

**Found on:**
Xcode: 6.1.
Mac OS X: 10.9.5.
Plugins: BAndroidPluginIndie-sbandroid-indie-6b3dd2094d160c74 or SBAndroidPluginProfessional-sbandroid-professional-046d84c81f2155f7.
SpriteBuilder Version 1.4.9.rc.0.

![screen shot 2015-04-06 at 4 01 10 pm](https://cloud.githubusercontent.com/assets/7450007/7003622/01af04ca-dc77-11e4-942d-2d0ae4cbebc8.png)

**Comment by ebrodzki** [comment]
This sample is meant to show off Android specific features. The mac and iOS targets are not relevant for this project

**Comment by khanbhai89** [comment]
@ebrodzki It happened with Android plugins as well while porting this project.

**Comment by netunlimited** [comment]
Indeed, I get the same error when I try to run this project on an actual Android device.
